rest_framework_json_api/serializers.py

```diff
@@ -152,8 +153,51 @@ def validate_path(serializer_class, field_path, path):
         super(IncludedResourcesValidationMixin, self).__init__(*args, **kwargs)
 
 
+class LazySerializersDict(Mapping):
+    def __init__(self, klass, serializers):
```

Comment:
when thinking about it `klass` is actually not really descriptive - I guess `parent` would be a better name?